### ISSUE TITLE
Provide list of reserved values (macros) in the documenation

### ISSUE THREAD

**Issue by gwfong** [arduino/Arduino#2394]
It appears that the macros PIN0, PIN1, PIN2, ..., PIN7 don't seem to work or represent their defined value. If I modify the standard Blink example:

#define PIN2 13

and use it throughout such as

digitalWrite(PIN2, HIGH);

it doesn't blink. I manually tried PIN0...PIN7. The LED doesn't blink when they are used. If these are reserved, I couldn't find anything that would tell me what they represent.

I'm currently using Arduino 1.0.6 64-bit on Ubuntu 14.04 64-bit running OpenJDK (IcedTea 2.5.3).

**Comment by cmaglie** [comment]
IMHO listing all the possible macros in the documentation is unmaintainable and not useful (who is going to look at such table before starting to program?).
Closing as wontfix.
